trust-store-trust-policy-specification.md

```diff
@@ -212,7 +207,7 @@ The trust policy is represented as JSON data structure as shown below:
   1. *Global*: If there exists a trust policy with global scope then use that policy for signature evaluation.
      Otherwise, fail the signature verification.
 
-### Trust Anchors Constraints
+### Trusted Identities Constraints
 
 A distinguished name (usually just shortened to "DN") uniquely identifies an entry and in the case of the certificate's subject, DN uniquely identifies the requestor/holder of the certificate.
 The DN is comprised of zero or more comma-separated components called relative distinguished names, or RDNs.
```

Comment:
in the line below do you mean to have '"' at the end of org1?

Reply:
Fixed in latest revision.